pulp_rpm/app/models.py

```diff
@@ -819,6 +843,29 @@ def libcomps_to_dict(cls, group):
             ),
         }
 
+    def pkg_grp_to_libcomps(self):
+        """
+        Convert PackageGroup object to libcomps Group object.
+
+        Returns:
+            group: libcomps.Group object
+
+        """
+        group = libcomps.Group()
+
+        group.id = getattr(self, PULP_GROUP_ATTRS.ID)
+        group.default = getattr(self, PULP_GROUP_ATTRS.DEFAULT)
+        group.uservisible = getattr(self, PULP_GROUP_ATTRS.USER_VISIBLE)
+        group.display_order = getattr(self, PULP_GROUP_ATTRS.DISPLAY_ORDER)
+        group.name = getattr(self, PULP_GROUP_ATTRS.NAME)
+        group.desc = getattr(self, PULP_GROUP_ATTRS.DESCRIPTION)
+        group.packages = self.list_to_pkglist(getattr(self, PULP_GROUP_ATTRS.PACKAGES))
+        group.biarchonly = getattr(self, PULP_GROUP_ATTRS.BIARCH_ONLY)
+        group.desc_by_lang = libcomps.StrDict(**getattr(self, PULP_GROUP_ATTRS.DESC_BY_LANG))
+        group.name_by_lang = libcomps.StrDict(**getattr(self, PULP_GROUP_ATTRS.NAME_BY_LANG))
```

Comment:
If you sync/publish  http://mirror.linux.duke.edu/pub/centos/7/os/x86_64/ , you will see that neither description, nor name by language are present in the output.

Reply:
We can address it as a bug fix in a follow-up PR.